client/views/admin/emailInbox/EmailInboxForm.js

```diff
@@ -158,9 +158,10 @@ export default function EmailInboxForm({ id, data }) {
 
 	const saveEmailInbox = useEndpointAction('POST', 'email-inbox');
 	const deleteAction = useEndpointAction('DELETE', `email-inbox?_id=${ id }`);
+	const emailAlreadyExistsAction = useEndpointAction('GET', id ? `email-inbox?_id=${ id }&email=${ email }` : `email-inbox?email=${ email }`);
```

Comment:
After creating a proper endpoint for searching emails, you will get the results here.